packages/compass-editor/src/codemirror/stage-autocompleter.ts

```diff
@@ -36,6 +32,15 @@ export const createStageAutocompleter = ({
         ? (['accumulator', 'accumulator:*'] as const)
         : []),
     ],
+  }).map((completion) => {
+    if (completion.meta?.startsWith('expr:')) {
+      return {
+        ...completion,
+        description: `<p>${aggLink(completion.value)} pipeline operator</p>`,
```

Comment:
"pipeline operator" is how they are referred to [in documentation](https://www.mongodb.com/docs/manual/reference/operator/aggregation/) even though we refer to them as expressions, don't really have a better idea for this description text